src/main/java/com/gmail/inverseconduit/bot/DefaultBot.java

```diff
@@ -52,6 +53,13 @@ private void processMessageQueue() {
     }
 
     private void processMessage(final ChatMessage chatMessage) {
+    	for (final CommandHandle listener : listeners){
+    		final String response = listener.execute(chatMessage);
+    		if (response != null){
```

Comment:
What does `findFirst` do?  Will that break out of the loop at the first occurrence of a non-null value?  If so, that's not what I want it to do.  I don't want it to break out of the loop...I want it to go through the entire collection.